prometheus/internalmetrics/internal_metrics.go

```diff
@@ -0,0 +1,202 @@
+// Package internalmetrics provides functionality to collect Prometheus metrics.
+package internalmetrics
+
+import (
+	"strconv"
+
+	"github.com/prometheus/client_golang/prometheus"
+	// Because this package is used all throughout the codebase, be VERY careful adding new
+	// kiali imports here. Most likely you will encounter an import cycle error that will
+	// cause a compilation failure.
+)
+
+// These constants define the different label names for the different metric timeseries
+const (
+	labelGraphKind        = "graph_kind"
+	labelGraphType        = "graph_type"
+	labelWithServiceNodes = "with_service_nodes"
+	labelAppender         = "appender"
+	labelName             = "name"
```

Comment:
This is a bit generic. Maybe `handler` or `route`?

Reply:
This is used for the kiali_api_processing_duration_seconds metric - I thought the name of the label `name` would be intuitively the name of the API. But perhaps `route` might be more intuitive?